atlp-rwanda/e-comm-team-techsmith-fn
# #185489217 Bug fix  for test on roles and permissions feature , notification feature and seller collections feature

## What does this PR do?
This PR fixes unit tests written for role and permissions seller collections and notification in generall
## Description of Task to be completed?

### This PR Should fix the following: 
- Admin should be able to disable an account for various reasons.
 - RBAC(Role based access control) - Admin should be able to assign roles [customer,merchant,admin]
 - Users should see a toast for new notifications
 - Notifications
 - Implement notifications pane
 - Marking all notifications as read
 - A seller should be able to update an product item from their collection
 - A seller should be able to delete a specific item from their collection

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout bg-unit-test-for-role-and-permissions-seller-collection-and-notification-185489217`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- Check out the components documentation: `npm run storybook`

## What are the relevant pivotal tracker/Trello stories?

- [#185489217](https://www.pivotaltracker.com/story/show/185489217)
